db/Controllers/news.controllers.js

```diff
@@ -20,4 +24,17 @@ const getApi = (request, response, next) => {
     });
 };
 
-module.exports = { getTopics, getApi };
+const getArticle = (request, response, next) => {
+  const article_id = request.params.article_id;
+
+  selectArticle(article_id)
+    .then((data) => {
+      //   console.log(data);
+      response.status(200).send(data);
```

Comment:
This should be an object with the key of article that contains an article object as a value.